graphql.schema.json

```diff
@@ -12,7 +12,6 @@
         "kind": "ENUM",
         "name": "ActiveStatus",
         "description": null,
-        "isOneOf": null,
```

Comment:
Can you try running `yarn` and then codegen again? I think these should be present.